src/dev/src/app/datagrid/binding-properties/binding-properties.html

```diff
@@ -16,20 +16,27 @@ <h2>Binding model properties to columns</h2>
 </p>
 <p>
   You can also see in the following example how every feature we offer is always opt-in: we did not declare any binding on
-  the "User ID" column, which means it is not sortable of filterable.
+  the "Favorite Color" column, which means it is not sortable of filterable.
+</p>
+
+<h4>Column types</h4>
+<p>
+  By default, bound columns are assumed to contain string-like contents and the user is presented with the normal string filter.
+  If you know that the contents of the column will be numeric, you can instead use the built-in numeric range filter by adding
+  <code class="clr-code">[clrDgColType]=&quot;'number'&quot;</code>.  You can see an example of this in the ID column.
 </p>
 
 <clr-datagrid>
-  <clr-dg-column>User ID</clr-dg-column>
-  <clr-dg-column [clrDgField]="'name'" [(clrFilterValue)]="nameFilter">Name</clr-dg-column>
   <clr-dg-column [clrDgField]="'creation'">Creation date</clr-dg-column>
+  <clr-dg-column [clrDgField]="'id'" [clrDgColType]="'number'">User ID</clr-dg-column>
```

Comment:
I forgot these weren't cleaned up. Do you mind using the "simpler" syntax without `[...]` binding?
For this column, it would be:
```html
<clr-dg-column clrDgField="id" clrDgColType="number">User ID</clr-dg-column>
```
It gets rid of these nasty nested quotes.